2-mandatory/1-create-functions.js

```diff
@@ -24,16 +26,22 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(arr) {
+  
+  return arr.map(str => str.trim().replace("/", "").toLowerCase());
 }
 
+
 /*
 Write a function that:
 - Takes an array and an index as input.
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, index) {
+  let newArray = arr.slice();
+  newArray.splice(index, 1);
```

Comment:
This will work, however it's neater to try and use the array methods `map, filter. reduce` whereever possible. In this case, this could be done with `.filter` via
```
function remove(arr, index) {
   return arr.filter((elem, idx) => idx !== index)
}
```